RetroBar/Languages/English.xaml

```diff
@@ -57,4 +57,8 @@
     <s:String x:Key="set_time">_Adjust Date/Time</s:String>
 
     <s:String x:Key="open_folder">&amp;Open Folder</s:String>
+
+    <FlowDirection x:Key="flow_direction">LeftToRight</FlowDirection>
+
+    <FontFamily x:Key="start_text_xp_font_family">Franklin Gothic</FontFamily>
```

Comment:
What's the reasoning behind this?

Reply:
Some other languages (such as Vietnamese and Hebrew) use Tahoma instead of Franklin Gothic as the start menu font for XP themes.

Reply:
Ah, interesting! I wonder if it makes sense for the font family itself to be defined in the language though--what if a theme wants to use different fonts than this but still swap fonts depending on language? Maybe another approach could be a bool value that indicates if an alternate font should be used, and the theme can use that to switch between them. Or, maybe the theme could switch fonts depending on the language name from settings?

Reply:
It is only for XP themes, that is defined, it is also fallback here in case it is not in the translation XAMLs.
The latter would no longer be dynamic, it would be defined behind the code, that's why we use dictionaries. We do not know when we need to use these fonts, except if it is hardcoded(?)

Reply:
I don't know if there is a better way, I leave it like this since users could change the font to the one they want for their language.

Reply:
I'm OK leaving it like this for now. We should revisit it later.